users/services/login.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.auth.models import User
+from django.core.exceptions import ObjectDoesNotExist
+from one_day_intern.exceptions import InvalidLoginCredentialsException
+from . import utils
+
+
+def verify_password(user: User, password: str):
+    if not user.check_password(password):
+        raise InvalidLoginCredentialsException(f'Password for {user.email} is invalid')
+
+
+def get_assessor_or_company_from_request_data(request_data):
+    try:
+        return utils.get_assessor_or_company_from_email(request_data.get('email'))
+    except ObjectDoesNotExist as exception:
+        raise InvalidLoginCredentialsException(exception)
+
+
+def login_assessor_company(request_data):
```

Comment:
Hello, Jo. I see that the login for the company and assessor has been made as of now. Will there be one for assessees as well?

Reply:
Hello, Alex. As per our discussion, there will be a dedicated end-point for assessee. However, the task is assigned to @indiradr. 

Qapla!

Reply:
Noted, Jo. I thought it was done together.

T!